scripts/load-breaches.js

```diff
@@ -41,12 +41,14 @@ async function handleBreachesResponse(response) {
 }
 
 (async () => {
-  if (args["--createAMBreach"]) {
-    await DBUtils.createBreach("AllMusic", {
-      Name: "AllMusic",
-      BreachDate: "2015-012-06",
+  if (args["--createExampleBreach"]) {
```

Comment:
can you remove this entire block? I think we can drop support for this example breach stuff at this point.

Reply:
yep! Do you want to take out --help too? And if not, what should we have the console.log on --help say?